nextflow.config

```diff
@@ -9,19 +9,30 @@
 // Global default params, used in configs
 params {
 
-    // TODO nf-core: Specify your pipeline's command line flags
     // Input options
     input                      = null
+    viral_fasta                = "https://data.broadinstitute.org/Trinity/CTAT_RESOURCE_LIB/VIRUS_INSERTION_FINDING_LIB_SUPPLEMENT/virus_db.nr.fasta"
```

Comment:
Could we have a possibility not to download it every time?

Reply:
We could add all the host + viraldb genomes added to aws igenomes, but that's a lot to maintain for now. 

Is there a reason you wouldn't want it downloaded every time? It's used in probably half the processes.

 I guess it could live in igenomes.config and put it with all of the human genomes, in terms of code wise, but I think this is easy for new users to find for now.

Reply:
I was thinking more of something like:
```
ctat_resource_base = `https://data.broadinstitute.org/Trinity/CTAT_RESOURCE_LIB/VIRUS_INSERTION_FINDING_LIB_SUPPLEMENT/`
viral_fasta = "${params.ctat_resource_base}/virus_db.nr.fasta"
```

Reply:
That could work, but it seems like an extra step? I guess the end user could use either:
```nextflow
viral_fasta = "./path/to/local/virus.fasta"
viral_fasta = "s3://somebucket/virus.fasta"
```
In that case, but I don't see the user ever modifying `ctat_resource_base`. As of right now, we're not downloading anything else, and the only thing they've changed is the file name! 😆

Reply:
yes, but also you could store it locally and run offline

Reply:
But in that case, wouldn't you just use `viral_fasta`? I'm imagining it would go in an institute config, but I don't see much difference between one line or two lines in this case. 

I think we should add it if we start having multiple resources to download, but at that point as well I think they should be in aws-igenomes or something we control.

Reply:
Good point.
I just feel like hardcoded URLs don't really belong in this file.